### ISSUE TITLE
WHERE

### ISSUE THREAD

**Issue by xinjirufen** [darold/ora2pg#1063]
Hi,darold
Dose the 'WHERE' directive support Chinese characters。I've tried it and found it doesn't support。
ora2pg.conf like this:
[ora2pg.txt](https://github.com/darold/ora2pg/files/5813339/ora2pg.txt)

**Comment by darold** [comment]
Do you have an error reported by Ora2Pg or just no row returned?

**Comment by xinjirufen** [comment]
> Do you have an error reported by Ora2Pg or just no row returned?

There was no error, just no data was exported

**Comment by xinjirufen** [comment]
> > Do you have an error reported by Ora2Pg or just no row returned?
> 
> There was no error, just no data was exported

My environment:
oracle 12c standard
ora2pg v21

**Comment by darold** [comment]
What does the following commands returns?
```
ora2pg -t SHOW_ENCODING -c config/ora2pg.conf
```
and
```
file config/ora2pg.conf
```
change the path to the configuration file following your installation.

**Comment by xinjirufen** [comment]
> What does the following commands returns?
> 
> ```
> ora2pg -t SHOW_ENCODING -c config/ora2pg.conf
> ```
> 
> and
> 
> ```
> file config/ora2pg.conf
> ```
> 
> change the path to the configuration file following your installation.

```
ora2pg -t SHOW_ENCODING -c config/ora2pg.conf
```
WARNING: target PostgreSQL version must be set in PG_VERSION configuration directive. Using default: 11
Current encoding settings that will be used by Ora2Pg:
	Oracle NLS_LANG AMERICAN_AMERICA.AL32UTF8
	Oracle NLS_NCHAR AL32UTF8
	Oracle NLS_TIMESTAMP_FORMAT YYYY-MM-DD HH24:MI:SS
	Oracle NLS_DATE_FORMAT YYYY-MM-DD HH24:MI:SS
	PostgreSQL CLIENT_ENCODING UTF8
	Perl output encoding ''
Showing current Oracle encoding and possible PostgreSQL client encoding:
	Oracle NLS_LANG AMERICAN_AMERICA.AL32UTF8
	Oracle NLS_NCHAR AL32UTF8
	Oracle NLS_TIMESTAMP_FORMAT YYYY-MM-DD HH24:MI:SS
	Oracle NLS_DATE_FORMAT YYYY-MM-DD HH24:MI:SS
	PostgreSQL CLIENT_ENCODING UTF8

```
file config/ora2pg.conf
```
UTF-8 Unicode text

**Comment by darold** [comment]
Yes, I have the same problem here with character with french accent. This is clearly an encoding issue, I don't know what Oracle is expecting through OCI but the comparison is failing. I have tried several encoding values for NLS_LANG or locale without success, finding the solution could take time you'd better use an other column with numbers or ANSI C characters, if possible of course.
